$NetBSD: patch-ae,v 1.1 2011/03/01 17:57:16 minskim Exp $

The "test" command, as well as the "[" command, are not required to know
the "==" operator.

--- test/indent/shell.sh.orig	2011-02-24 08:45:25.000000000 +0000
+++ test/indent/shell.sh
@@ -70,7 +70,7 @@ help2
 EOF2
 }
 bar () {
-    if [ $# == 0 ]; then
+    if [ $# = 0 ]; then
         while
             f                   # KNOWN INDENT BUG
         do
